diff --git a/node_modules/@nuxt/content/dist/runtime/components/ContentRenderer.vue b/node_modules/@nuxt/content/dist/runtime/components/ContentRenderer.vue
index 1210c7b..18e41cc 100644
--- a/node_modules/@nuxt/content/dist/runtime/components/ContentRenderer.vue
+++ b/node_modules/@nuxt/content/dist/runtime/components/ContentRenderer.vue
@@ -209,7 +209,7 @@ function findMappedTag(node: MDCElement, tags: Record<string, string>) {
     :key="key"
     :body="body"
     :data="data"
-    :clas="props.class"
+    :class="props.class"
     :tag="props.tag"
     :prose="props.prose"
     :unwrap="props.unwrap"
